about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Exhibition</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <ul class="top-navbar-menu">
+                    <li><a href="">Facebook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Log out</a></li>
+                </ul>
+                <div class="bottom-navbar">
+                    <a class="logo" href="index.html">Logo</a>
+                    <ul class="bottom-navbar-menu">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="">Program</a></li>
+                        <li><a href="">Join</a></li>
+                        <li><a href="">Sponsor</a></li>
+                        <li><a href="">News</a></li>
+                        <li><a href="">Campiagn</a></li>
+                    </ul>   
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="about-main-section primary-font">
+            <div class="about-info container">
+                <button class="nav-btn"><i class="fa-solid fa-bars fa-2x"></i></button>
+                <p class="sub-title">"Kabul Mesuem Exhibition"</p>
+                <h1 class="title">FOURTH ROUND OF KABUL MESUEN EXHABITION</h1>
+                <hr class="title-underline">
+                <div class="about-short-info">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer congue mollis malesuada. Mauris enim massa, interdum dictum vestibulum at, pretium vitae libero. Morbi libero tortor, luctus tincidunt purus et, elementum tempus nibh. Etiam elit nisl, sagittis quis consequat sit amet, congue vitae turpis. In pellentesque aliquam felis eu varius. Pellentesque interdum nisl ut justo tempor tincidunt. Ut sodales vel quam a semper.</p>
+                    <p>Duis accumsan venenatis eros. Nullam interdum neque lectus, eget pretium ligula vulputate nec. Proin et ipsum pharetra, lobortis est ut, suscipit nisi. Mauris blandit laoreet nibh. Mauris semper, orci ac iaculis egestas, magna dolor suscipit justo, varius vehicula purus quam quis enim. In consectetur pulvinar dui, ut vestibulum orci rhoncus et. Aliquam nec lacinia lacus.</p>
```

Comment:
- [ ] This paragraph should have proper padding as seen below 👇🏾 

Design guideline
![image](https://user-images.githubusercontent.com/84156714/233768496-32dafc7f-2575-44d0-a530-9239e9526bfd.png)

Your work
![image](https://user-images.githubusercontent.com/84156714/233768451-8f7d9cc5-71c0-492e-b0a3-19284ce5b289.png)